contracts/RandomGenerator.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.6.0;
+
+import "https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/VRFConsumerBase.sol";
+
+contract RandomGenerator is VRFConsumerBase  {
```

Comment:
Maintain roomId => randomNo mapping and requestId => roomid mapping here.

Reply:
Do we need requestId mapping ?

Reply:
roomId=>randomNo mapping already exists in GameCoin contract where its being used, any specific reason to move that here ?

Reply:
Yes, if you don't there will be dirty update problem.